src/com/visualization/barchart/bar-axis-tick.js

```diff
@@ -0,0 +1,16 @@
+import {h, Component} 				from 'preact/preact';
+
+const MAJOR_TICK_LENGTH = 1.5;
+const MINOR_TICK_LENGTH = 0.5;
+
+export default class Tick extends Component {
+	render({major, horizontal, outside, axis, position}) {
```

Comment:
```js
render( props ) {
    var {major, horizontal, outside, axis, position} = props;
```

Reply:
I'm going to hope I can still use `let` and `const` right?